src/librustc_mir/hair/pattern/_match.rs

```diff
@@ -618,8 +619,12 @@ fn max_slice_length<'p, 'a: 'p, 'tcx: 'a, I>(
 }
 
 /// An inclusive interval, used for precise integer exhaustiveness checking.
-/// `Interval`s always store a contiguous range of integers. That means that
-/// signed integers are offset (see `offset_sign`) by their minimum value.
+/// `Interval`s always store a contiguous range of integers. This means that
+/// signed values are encoded by offsetting them such that `0` represents the
```

Comment:
"by offseting them" is potentially confusing. Also, "encodes" instead of "represents" would help.
And actually, the minimum value thing applies to all integers, regardless of sign*edness*.